### PR TITLE
[zh] Translate /docs/reference/kubernetes-api/authentication-resource…

### PR DESCRIPTION
**Pull request by javadoors**
[zh] Translate /docs/reference/kubernetes-api/authentication-resources/certificate-signing-request-v1/ to Chinese https://github.com/kubernetes/website/issues/34137

Add the page: /zh/docs/reference/kubernetes-api/authentication-resources/certificate-signing-request-v1/

### CODE REVIEW

File: content/zh-cn/docs/reference/kubernetes-api/authentication-resources/certificate-signing-request-v1.md [hunk 1]
```diff
@@ -0,0 +1,1600 @@
...
+  *Atomic: will be replaced during a merge*
+  
+  request contains an x509 certificate signing request encoded in a "CERTIFICATE REQUEST" PEM block. 
+  When serialized as JSON or YAML, the data is additionally base64-encoded.
+-->
+<hr>
+
+- **request** ([]byte)，必需
+
+  *Atomic：将在合并期间被替换*
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/authentication-resources/certificate-signing-request-v1.md:138]: 此类单星号都要在翻译后换成双星号，从斜体改为加粗：

```suggestion
  **Atomic：将在合并期间被替换**
```

File: content/zh-cn/docs/reference/kubernetes-api/authentication-resources/certificate-signing-request-v1.md [hunk 2]
```diff
@@ -0,0 +1,1600 @@
...
+   "key encipherment", "key agreement", "data encipherment",
+   "cert sign", "crl sign", "encipher only", "decipher only", "any",
+   "server auth", "client auth",
+   "code signing", "email protection", "s/mime",
+   "ipsec end system", "ipsec tunnel", "ipsec user",
+   "timestamping", "ocsp signing", "microsoft sgc", "netscape sgc"
+-->
+  有效值：
+   "signing"、"digital signature"、"content commitment"、
+   "key encipherment"、"key agreement"、"data encipherment",
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/authentication-resources/certificate-signing-request-v1.md:314]:
```suggestion
   "key encipherment"、"key agreement"、"data encipherment"、
```

**Review by tengqm — COMMENTED** [review]
A few nits for you to practice "git commit --amend; git push -f".

File: content/zh-cn/docs/reference/kubernetes-api/authentication-resources/certificate-signing-request-v1.md (hunk 2 above)
Comment (javadoors) [reply content/zh-cn/docs/reference/kubernetes-api/authentication-resources/certificate-signing-request-v1.md]: How careless I am。

**Comment by javadoors** [comment]
I amend it again.

**Pull request merged**